framework: ROS

horizon_duration: 1.5
n_nodes: 31

fixed_joints:
    "arm_left_5_joint": 0.004009140644162072
    "arm_left_6_joint": 0.00358625686761845
    "arm_left_7_joint": 0.02561515245675291
    "arm_right_5_joint": 0.004009140644162072
    "arm_right_6_joint": 0.00358625686761845
    "arm_right_7_joint": 0.02561515245675291
    "head_1_joint": -0.027120105662825997
    "head_2_joint": -0.0002205888117432746

define:
    # open loop
    - &low_stiff 500.
    - &low_damp 60.
    - &high_stiff 2000.
    - &high_damp 100.

    # closed loop
#    - &low_stiff 50.
#    - &low_damp 10.
#    - &high_stiff 200.
#    - &high_damp 10.

stiffness:
    "leg_left_1_joint": *high_stiff
    "leg_left_2_joint": *high_stiff
    "leg_left_3_joint": *high_stiff
    "leg_left_4_joint": *high_stiff
    "leg_left_5_joint": *high_stiff
    "leg_left_6_joint": *high_stiff
    "leg_right_1_joint": *high_stiff
    "leg_right_2_joint": *high_stiff
    "leg_right_3_joint": *high_stiff
    "leg_right_4_joint": *high_stiff
    "leg_right_5_joint": *high_stiff
    "leg_right_6_joint": *high_stiff
    "torso_1_joint": *high_stiff
    "torso_2_joint": *high_stiff
    "arm_left_1_joint": *low_stiff
    "arm_left_2_joint": *low_stiff
    "arm_left_3_joint": *low_stiff
    "arm_left_4_joint": *low_stiff
    "arm_left_5_joint": *low_stiff
    "arm_left_6_joint": *low_stiff
    "arm_left_7_joint": *low_stiff
    "arm_right_1_joint": *low_stiff
    "arm_right_2_joint": *low_stiff
    "arm_right_3_joint": *low_stiff
    "arm_right_4_joint": *low_stiff
    "arm_right_5_joint": *low_stiff
    "arm_right_6_joint": *low_stiff
    "arm_right_7_joint": *low_stiff
    "head_1_joint": *low_stiff
    "head_2_joint": *low_stiff
#
damping:
    "leg_left_1_joint": *high_damp
    "leg_left_2_joint": *high_damp
    "leg_left_3_joint": *high_damp
    "leg_left_4_joint": *high_damp
    "leg_left_5_joint": *high_damp
    "leg_left_6_joint": *high_damp
    "leg_right_1_joint": *high_damp
    "leg_right_2_joint": *high_damp
    "leg_right_3_joint": *high_damp
    "leg_right_4_joint": *high_damp
    "leg_right_5_joint": *high_damp
    "leg_right_6_joint": *high_damp
    "torso_1_joint": *high_damp
    "torso_2_joint": *high_damp
    "arm_left_1_joint": *low_damp
    "arm_left_2_joint": *low_damp
    "arm_left_3_joint": *low_damp
    "arm_left_4_joint": *low_damp
    "arm_left_5_joint": *low_damp
    "arm_left_6_joint": *low_damp
    "arm_left_7_joint": *low_damp
    "arm_right_1_joint": *low_damp
    "arm_right_2_joint": *low_damp
    "arm_right_3_joint": *low_damp
    "arm_right_4_joint": *low_damp
    "arm_right_5_joint": *low_damp
    "arm_right_6_joint": *low_damp
    "arm_right_7_joint": *low_damp
    "head_1_joint": *low_damp
    "head_2_joint": *low_damp

